extension.js

```diff
@@ -273,27 +273,60 @@ async function build(config, componentPath, server) {
 		}
 
 		// Build
-		const [command, ...args] = shellQuote.parse(config.buildCommand);
+		try {
+			logger.info(`Building Next.js application at ${componentPath}`);
+
+			const timerStart = performance.now();
+
+			const stdout = [];
+			const stderr = [];
+
+			const buildProcess = spawn(config.buildCommand, [], {
+				shell: true,
+				cwd: componentPath,
+				stdio: ['ignore', 'pipe', 'pipe'],
+			});
+
+			buildProcess.stdout.pipe(
```

Comment:
Do you need a writable stream? Child process `stdout` and `stderr` are event emitters that emit a `data` event with a buffer containing the output.

```
buildProcess.stdout.on('data', s=> stdout.push(s.toString()));
```